app/marketing/mails.py

```diff
@@ -686,6 +685,33 @@ def weekly_recap(to_emails=None):
         finally:
             translation.activate(cur_language)
 
+def unread_notification_email_weekly_roundup(to_emails=None):
+    if to_emails is None:
+        to_emails = []
+    
+    for to_email in to_emails:
+        cur_language = translation.get_language()
+        try:
+            setup_lang(to_email)
+            html, text, subject = render_unread_notification_email_weekly_roundup(to_email)
+            from_email = settings.PERSONAL_CONTACT_EMAIL
+
+            if not should_suppress_notification_email(to_email, 'weeklyrecap'):
+                send_mail(
+                    from_email, 
+                    to_email, 
+                    subject, 
+                    text, 
+                    html, 
+                    from_name="Kevin Owocki (Gitcoin.co)", 
```

Comment:
```
from_name="Kevin Owocki (Gitcoin.co)"
categories=['marketing', func_name()]
```

Above value are repeated 3 times could you create const variables at the top of file.

Reply:
Can you elaborate on this please

Reply:
He means the two values are used in multiple places , so he's suggesting we move it a constant and just reuse the constant.
I'm good either way so I leave it to you @iamonuwa to make the call

Reply:
Am fine with leaving as it is.